templates/stripes/stripe-table.php

```diff
@@ -0,0 +1,55 @@
+<?php
+$caption = !empty($caption["html"]) ? $this->inline($caption["html"]) : null;
+$headersInFirstRow = !empty($headersInFirstRow) ? $headersInFirstRow : false;
+$headersInFirstColumn = !empty($headersInFirstColumn) ? $headersInFirstColumn : false;
+
+// Ensure each item in the dataset is an array.
+$dataset = !empty($dataset) && is_array($dataset) ? $dataset : [[]];
+foreach ($dataset as $key => $value) {
+    if (!is_array($value)) {
+        $dataset[$key] = [];
+    }
+}
+?>
+
+<div class="<?= $this->e($type, 'wrapperClasses') ?>">
+  <div class="vssl-stripe-column">
+    <table>
+      <?php if (!empty($caption)) : ?>
+        <caption><?= $caption ?></caption>
+      <?php endif; ?>
+
+      <?php if ($headersInFirstRow && !$headersInFirstColumn): ?>
+        <thead>
+          <?php foreach ($dataset[0] as $index => $item): ?>
+            <th><?= $item ?></th>
+          <?php endforeach; ?>
+        </thead>
+      <?php endif; ?>
+
+      <tbody>
+        <?php if ($headersInFirstRow && $headersInFirstColumn): ?>
```

Comment:
This is the same content as the condition above, right? You can just do one if (`$headersInFirstRow`)?

Reply:
(This isn't true)